extensions/panache/mongodb-panache/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -66,6 +67,11 @@ void registerJacksonSerDeser(BuildProducer<JacksonModuleBuildItem> customSerDese
                         .build());
     }
 
+    @BuildStep
+    NativeImageResourceBuildItem registerForRessource() {
+        return new NativeImageResourceBuildItem("application.properties");
+    }
+
```

Comment:
You should not need this

Reply:
Indeed, what was the rationale for adding this?

Reply:
It was because on panache integration test with native build do not succeed to get config of Mongo database. This is the only way I found to force load of config for panache

Reply:
Then there must be something else going on, this is almost certainly not the proper approach.
An extension shouldn't need access to the `application.properties` resource, but should instead read the configuration using MP Config

Reply:
issue come from MongoOperations.java with 
```
String databaseName = ConfigProvider.getConfig()
                .getValue(MONGODB_DATABASE, String.class);
```
Config comming from application.properties is not worling on native mode... I was thinking it is because it is loaded in another module and not panache. But what is solution to have it on panache ?